docs/releases/4.0.txt

```diff
@@ -261,6 +261,12 @@ Tests
 * Django test runner now supports a :option:`--buffer <test --buffer>` option
   with parallel tests.
 
+* :meth:`~django.test.SimpleTestCase.assertHTMLEqual` now handles
+  `boolean attributes <https://html.spec.whatwg.org/#boolean-attribute>`_
+  more correctly so it knows that ``<input checked="">`` is the same as
+  ``<input checked="checked">`` but that ``<input value="">`` is not the same
+  as ``<input value="value">``.
```

Comment:
I don't think we need release notes, maybe a small note in the ["Backwards incompatible changes in 4.0"](https://docs.djangoproject.com/en/dev/releases/4.0/#miscellaneous) section.